packages/block-library/src/query-pagination-numbers/edit.js

```diff
@@ -19,7 +19,13 @@ const previewPaginationNumbers = () => (
 	</>
 );
 
-export default function QueryPaginationNumbersEdit() {
+export default function QueryPaginationNumbersEdit( attributes ) {
+	const blockProps = useBlockProps();
```

Comment:
Not mandatory, but it's also possible to pass the gap style to `useBlockProps` so we don't have to destructure the styles inline.

```jsx
export default function QueryPaginationNumbersEdit( { attributes } ) {
	const blockProps = useBlockProps( {
		style: {
			gap: attributes?.style?.spacing?.blockGap,
		},
	} );
	return <div { ...blockProps }>{ previewPaginationNumbers() }</div>;
}
```